tests/python/gpu/test_operator_gpu.py

```diff
@@ -2319,3 +2319,21 @@ def test_fp16_spmm():
     out = mxsps.dot(inp, weight)
     out_np = mx.nd.dot(inp, weight)
     assert_almost_equal(out.asnumpy(), out_np, rtol=1e-3, atol=1e-5)
+
+@with_seed()
+@pytest.mark.serial
```

Comment:
no need to mark as serial

Reply:
I noticed that the `mark.serial` is triggered for all tests in this file. Thus, we may keep `mark.serial`.

Reply:
We run tests based on the tag and it has nothing to do with file. serial is only needed when test invocation is long-running and consumes lots of memory. Since this is no longer the case through parametrizing the input, the serial tag is not needed.

please open a follow up PR to finish the change.